src/irr/asset/IAssetManager.cpp

```diff
@@ -392,4 +455,18 @@ void main()
         addBuiltInToCaches(ds1, "irr/builtin/descriptor_set/basic_view_parameters");
         addBuiltInToCaches(defaultDs1Layout, "irr/builtin/descriptor_set_layout/basic_view_parameters");
     }
+
+    // pipeline layout
+    core::smart_refctd_ptr<asset::ICPUPipelineLayout> pipelineLayout;
+    {
+        asset::ICPUDescriptorSetLayout::SBinding bnd;
+        bnd.count = 1u;
+        bnd.binding = 0u;
+        bnd.stageFlags = static_cast<asset::ICPUSpecializedShader::E_SHADER_STAGE>(asset::ICPUSpecializedShader::ESS_VERTEX | asset::ICPUSpecializedShader::ESS_FRAGMENT);
+        bnd.type = asset::EDT_UNIFORM_BUFFER;
+        auto ds1Layout = core::make_smart_refctd_ptr<asset::ICPUDescriptorSetLayout>(&bnd, &bnd + 1);
+
+        pipelineLayout = core::make_smart_refctd_ptr<asset::ICPUPipelineLayout>(nullptr, nullptr, nullptr, std::move(ds1Layout), nullptr, nullptr);
+        addBuiltInToCaches(pipelineLayout, "irr/builtin/materials/lambertian/no_texture/pipelinelayout");
```

Comment:
add an alias `irr/builtin/loaders/PLY/pipelinelayout` (you can cache under multiple paths)